src/ui/graph.rs

```diff
@@ -1,69 +1,104 @@
-use std::collections::{HashMap, HashSet};
-
 use egui::Widget;
 use egui_nodes::{LinkArgs, NodeArgs, NodeConstructor, PinArgs};
+use std::collections::{HashMap, HashSet};
+
+use super::Theme;
+use crate::pipewire_impl::{MediaType, PortType};
 
-use crate::pipewire_impl::MediaType;
+#[derive(Debug)]
```

Comment:
Moving ```Node```, ```Link```, ```Port``` into the this module might not a good idea for organization sake, I plan on adding more code to these modules like [here](https://github.com/Ax9D/pw-viz/blob/grouped_nodes/src/ui/node.rs).
So i suggest leaving them in their own files.

Reply:
Ok, should be resolved.